.gitignore

```diff
@@ -27,3 +27,4 @@ GRTAGS
 GTAGS
 compile_commands.json
 config/
+documentation/
```

Comment:
What is this dir for actually?

Reply:
This directory is where the documentation is going to be built. I want it to be ignored so it doesn't end up on the repository.